react/features/face-centering/reducer.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+
+import { ReducerRegistry } from '../base/redux';
+
+import {
+    START_FACE_RECOGNITION,
+    STOP_FACE_RECOGNITION,
+    UPDATE_FACE_COORDINATES
+} from './actionTypes';
+
+const defaultState = {
+    faceBoxes: {},
```

Comment:
doc: Could you please add some docs? Especially description for the format of the data in `faceBoxes` will be very helpful. For example I'm having a hard time to understand what the percentage value stored in `left`, `right`, `top` and `bottom` is representing. 

If we get `left` for example. I guess I can convert it to pixels if I apply the percent to the width.But then is this the distance from the left edge of the video to the left edge of the faceBox or something else?